libraries/cyclestreets-view/src/main/res/values-fr/strings.xml

```diff
@@ -63,6 +63,8 @@
   <string name="report_title">Votre rapport</string>
   <string name="report_location_hint">Où est l\'emplacement de votre rapport décrit?</string>
 
+  <string name="tap_here">Appuyez ici pour voir le site Web</string>
```

Comment:
All these translations need updating please

Reply:
Done - I was holding off in case of any change to the text :-)